packages/palette/src/elements/Checkbox/Check.tsx

```diff
@@ -68,7 +68,7 @@ export const Check: React.FC<CheckProps> = ({
       selected={selected}
       {...rest}
     >
-      <CheckIcon fill={iconColor()} />
+      <CheckIcon aria-hidden fill={iconColor()} />
```

Comment:
Caught this when doing a voiceover pass. All our SVGs have names so this was calling out "Check" here.